guix-science/packages/zotero.scm

```diff
@@ -0,0 +1,63 @@
+                     "/Zotero-" version "_linux-x86_64.tar.bz2"))
+               (sha256
+                (base32
+                 "0h1bizcqlk22h7nvhnyp3ppymv2hrk7133hgmp15hl3bvfzz7nh6"))))
+      (build-system binary-build-system)
+      (arguments
+       `(#:patchelf-plan `(("zotero-bin" ("gcc:lib"))
+                           ("libmozgtk.so" ("gtk+"))
+                           ("libxul.so" ("libx11" "dbus-glib" "libxt" "libgthread")))
+         #:install-plan '(("./" "/opt/zotero/"))
+         #:phases (modify-phases %standard-phases
+                                 (add-after 'patch-usr-bin-file 'patch-launcher
+                                            (lambda* (#:key outputs #:allow-other-keys)
+                                              (substitute* "zotero"
+                                                           ((,(regexp-quote
+                                                               "$(dirname \"$(readlink -f \"$0\")\")"))
+                                                            (string-append (assoc-ref outputs "out")
+                                                                           "/opt/zotero")))))
+                                 (add-after 'install 'install-bin
+                                            (lambda* (#:key outputs #:allow-other-keys)
+                                              (mkdir-p (string-append (assoc-ref outputs "out") "/bin"))
+                                              (symlink (string-append (assoc-ref outputs "out")
+                                                                      "/opt/zotero/zotero")
+                                                       (string-append (assoc-ref outputs "out")
+                                                                      "/bin/zotero"))
+                                              (mkdir-p (string-append (assoc-ref outputs "out")
+                                                                      "/share/applications"))
+                                              (symlink (string-append (assoc-ref outputs "out")
+                                                                      "/opt/zotero/zotero.desktop")
+                                                       (string-append (assoc-ref outputs "out")
+                                                                      "/share/applications/zotero.desktop")))))))
+      (inputs `(("coreutils" ,coreutils)
+                ("gcc:lib" ,gcc "lib")
+                ("gtk+" ,gtk+)
+                ("libx11" ,libx11)
+                ("libgthread", glib)
+                ("libxt" ,libxt)
+                ("dbus-glib" ,dbus-glib)))
+      (synopsis
+       "Your personal research assistant.")
+      (description
+       "Zotero is a free, easy-to-use tool to help you collect, organize, annotate, cite, and share research.")
+      (home-page "https://github.com/latex-lsp/texlab")
+      (license license:gpl3)))
```

Comment:
According to https://github.com/zotero/zotero/blob/main/COPYING the license is actually AGPL 3.

Reply:
Typo, fixed.